### ISSUE TITLE
Not matching with Trakt.

### ISSUE THREAD

**Issue by olliebean** [tegon/traktflix#140]
As of today, nothing is being identified on Trakt - everything is showing "Not found on Trakt". I tried manually pasting a Trakt URL into one item, but it gave the message "An error happened, please make sure you are entering a valid URL and try again". Is it just a matter of waiting until it starts working again, or does something need fixing?

**Comment by elemis** [comment]
I have the same issue.

**Comment by olliebean** [comment]
Still not working today. I think Trakt must have made some change that has broken it.

**Comment by rafaelgssa** [comment]
I'll see if I can take a look today.

**Comment by olliebean** [comment]
Any progress/insight into this yet? Still having the same issue, although other apps I have that interface with Trakt are working fine.

**Comment by rafaelgssa** [comment]
I tested on Firefox and everything matched fine for me. Are you on Chrome? I'll see if I can run more tests this week.

**Comment by olliebean** [comment]
No, I'm on Firefox. I'm a couple of versions behind, but the addon hasn't been updated in months and was working fine until a week ago.

I also tried it in a clean Firefox profile, and in Chrome, with the same result in each case. Could it be a regional issue? I'm in the UK. Perhaps elemis could comment if they are also outside the US. Not sure why that should make a difference, though.

Incidentally, there's another problem in recent versions of Firefox, which is that the addon's icon doesn't show - luckily I'd previously bookmarked the sync page, as without the icon there's no other way to get to it.

**Comment by elemis** [comment]
I'm on Firefox too. Chrome had this issue for a long time now, so I've been using Firefox for traktflix ever since. I'm in Turkey. A week ago Firefox worked fine.

**Comment by olliebean** [comment]
FWIW, I tried running a VPN to put myself into the US region, and the addon spun it's wheel for a lot longer, but ultimately still didn't manage to find any Trakt matches.

Does the addon produce any logs that we could look at to get a clue what the problem is?

**Comment by olliebean** [comment]
The debugging console shows twenty 401 errors (i.e., one for each title on the page) associated with history-sync.js when I load the sync page, which suggests it's an authentication issue.

**Comment by rafaelgssa** [comment]
Do you use Firefox containers?

**Comment by olliebean** [comment]
I only use the Facebook Container addon, which shouldn't affect Trakt or Netflix. I have Cookie AutoDelete installed, but Trakt and Netflix are whitelisted. In any case, neither of these were installed in the clean profile I tried it in.

I have also tried turning off the blocking in Firefox's Enhanced Tracking Protection, which didn't help.

How do I log out and log in again in the extension? I don't see the option anywhere - if it's in the dropdown from the icon, I can't access it as the icon is missing, as mentioned in an earlier comment. I did try clearing storage and Trakt cache from the addon's options page, but that didn't help.

**Comment by olliebean** [comment]
OK, I revoked access using the link @smoothlystable gave, but then I couldn't login again, because the login button is in the icon popup, and I have no icon. But I discovered by examining the contents of the addon that the popup is contained in `popup.html`, so I was able to access that by replacing the filename in my history sync bookmark, which allowed me to login to Trakt again via the addon, and now it seems to be working again.

In response to the last message from @rafaelgssa, I am on v2.1.1 since January - do you think it was an expired token that caused this? In which case, maybe the bug persists...

**Comment by rafaelgssa** [comment]
@ollibean Do you know how to access the console log from the background page of the extension? You should be able to do it by going to `about:debugging#/runtime/this-firefox` and clicking `Inspect` next to the traktflix extension. Are there any errors there that could be related to the icon not appearing?

**Comment by olliebean** [comment]
@rafaelgssa Yes, I already looked there; that was where I saw the 401 errors that I mentioned in an earlier comment. I can't see anything else there that obviously relates to this addon, though. What specific types of error should I be looking for?

**Comment by olliebean** [comment]
I tried again with a clean profile, so traktflix was the only addon installed, and the only message that appeared in the console log for traktflix after restarting the browser was this:

`Could not map contract ID '@mozilla.org/sharepicker;1' to CID {1201d357-8417-4926-a694-e6408fbedcf8} because no implementation of the CID is registered.`

The icon was not present, in the toolbar or on the Customise page.

**Comment by rafaelgssa** [comment]
I don't think the icon appears in the customize page, it only appears for Netflix pages. I'll do some tests here. Thanks for the info.
